tests/users/test_api_get_user_details.py

```diff
@@ -0,0 +1,148 @@
+import unittest
+from http import HTTPStatus, cookies
+from unittest.mock import patch, Mock
+import requests
+from requests.exceptions import HTTPError
+from flask import json
+from flask_restx import marshal
+from app import messages
+from tests.base_test_case import BaseTestCase
+from app.api.request_api_utils import post_request, BASE_MS_API_URL, AUTH_COOKIE
+from app.api.resources.users import MyProfilePersonalDetails
+from app.api.models.user import full_user_api_model
+from tests.test_data import user1
+
+
+class TestGetUserDetailsApi(BaseTestCase):
+    
+    @patch("requests.get")
+    @patch("requests.post")
+    def test_api_get_user_details_with_correct_token(self, mock_login, mock_get_user):
+        success_message = {"access_token": "this is fake token", "access_expiry": 1601478236}
```

Comment:
Can you add a comment about that access_expiry? Is that past? future, what is the date represented there?

Reply:
done, @isabelcosta . Now also with changed commit message as we discussed in the meeting before 😉